### ISSUE TITLE
Add other installation options

### ISSUE THREAD

**Issue by brumhard** [brumhard/krewfile#1]
To make installation easier some of the following options could be added:
- release as krew plugin (just like krew itself is a krew plugin)
- create binaries for all platforms
- create OCI images
- create linux packages
- create homebrew formula

**Comment by halkeye** [comment]
go releaser would do a bunch of those for you

**Comment by brumhard** [comment]
@halkeye yup, just need to copy it from other repos. Would you benefit from those additional sources or do you also use nix?

**Comment by halkeye** [comment]
I don't use nix. But also when I realized it's just calling krew install I might just do the same via my ansible scripts. Not sure yet
